### DISCUSSION TITLE
General Complete Layout

### DISCUSSION THREAD

**Discussion by DataHawker** [fuma-nama/fumadocs#805] [General]
I'm looking for a basic complete next app (not turbopack) with the same structure as the fumadocs main. I like the first-page header, the config for the sidebars, and the TOC. I want to host multiple technical docs on one site and I'm having difficulty structuring this project.

**Comment by fuma-nama** [comment]
You can use Home Layout. To share the layout across multiple pages you can create a Route Group. See docs for details

**Comment by Gitstar-OC** [comment]
You can just clone this repository and learn how to host multiple docs, it will help. You are welcome!!

**Comment by DataHawker** [comment]
I'm struggling with the mono repo. I have a few issues where I'm following the docs but can't figure out why it's not working. 

1. The top nav is showing even on the doc pages.
2. Can't figure out the best way to have many separate docs. I want this to be a digital library for technical documents for my consulting job. I can create mdx files without issues but bucket them into their own root folders, i'm having a hard time.
3. I also want to add items to the navbar like profile aviator to bring up logout, profile, and in the sidebar, the theme. 

I am willing to pay a bit for some 1 on 1 help getting this setup.

**Reply by fuma-nama** [reply]
> The top nav is showing even on the doc pages.

You need to add the HomeLayout to another route group instead of your root layout.tsx, otherwise it impacts the other layouts. It is same as Next.js routing, Make sure to read their docs for fundamentals 

> I can create mdx files without issues but bucket them into their own root folders, i'm having a hard time.

Make a folder with meta.json inside, add `root: true` to set it as a root folder. You can put your MDX files there, then add the root toggle to your sidebar or other places you want.

> I also want to add items to the navbar like profile aviator to bring up logout, profile, and in the sidebar, the theme.

Check Links API, it handles that

**Comment by Gitstar-OC** [comment]
I can help you, on that and do that would you like to connect with me, I can help you with all of this things, also you can check my [docs](https://learnai.theme-verse.com) that I have made, should I do the setup for you ?

**Comment by DataHawker** [comment]
Yes please - email?
